phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewIT.java

```diff
@@ -387,6 +388,48 @@ public void testViewUsesTableLocalIndex() throws Exception {
         }
     }
 
+    @Test
+    public void testCreateViewTimestamp() throws Exception {
+        String tenantId = null;
+        createViewTimestampHelper(tenantId);
```

Comment:
I could just pass null, but this seemed more self-documenting (because the name makes it clear that I'm passing a null tenant id, rather than making someone need to look at the function def to figure that out)